src/murrApp.vue

```diff
@@ -89,9 +84,16 @@
             }
         },
 
+        watch: {
+            '$route'() {
+                // eslint-disable-next-line no-console
+                this.murr_btn_clicked = false;
+            }
+        },
+
         methods: {
             click_murr_btn() {
-                return this.murr_btn_clicked = !this.murr_btn_clicked;
```

Comment:
**Never write return in methods, they are written exclusively in the computed properties!!!**